web/atlas.json

```diff
@@ -3784,7 +3782,7 @@
 {"id": "tx41jb", "name": "Yuzusoft and Murasame", "description": "Operation YuzuArt--This art created by Yuzusoft Fan Server on discord.", "links": {"discord": ["jJRNgzh"]}, "path": {"56-166, T": [[1426, 508], [1472, 508], [1472, 502], [1432, 502], [1432, 492], [1426, 492]]}, "center": {"56-166, T": [1445, 504]}},
 {"id": "tx41gk", "name": "Blue portal", "description": "A blue portal, a reference to the Portal games. It was built by the Germans and the Portal community to connect the split flags after the first canvas expansion because they didn't want to destroy the artworks in between.", "links": {"website": ["https://www.thinkwithportals.com/"], "subreddit": ["placeDE", "Portal"]}, "path": {"56-165, T": [[1067, 829], [1066, 828], [1064, 828], [1062, 829], [1061, 831], [1060, 833], [1059, 838], [1058, 839], [1057, 845], [1057, 850], [1058, 853], [1059, 854], [1060, 856], [1061, 860], [1062, 863], [1069, 869], [1063, 869], [1062, 868], [1061, 867], [1060, 865], [1058, 863], [1057, 861], [1056, 857], [1056, 855], [1056, 845], [1064, 863], [1062, 861]]}, "center": {"56-165, T": [1063, 825]}},
 {"id": "tx41dx", "name": "Inky", "description": "A small version of the famous ghost Inky in the video game Pac-Man. Made by u/Ghostieclone and friends.", "links": {"website": ["https://en.wikipedia.org/wiki/Ghosts_(Pac-Man)"]}, "path": {"109-165, T": [[1041, 1115], [1038, 1115], [1036, 1115], [1036, 1120], [1041, 1120]]}, "center": {"109-165, T": [1039, 1118]}},
-{"id": "tx40ya", "name": "2balkan4you", "description": "In memory of the subbredit 2balkan4you banned by u/Chtorrr in the beginning 2022, where people from the balkans could joke together.\nMany new subreddits have tried to reconstruct the community of around 200k members, but the majority of them are empty.", "links": {"website": ["https://subredditstats.com/r/2balkan4you"]}, "path": {"56-166, T": [[1918, 322], [1918, 334], [1936, 334], [1936, 322]]}, "center": {"56-166, T": [1927, 328]}},
+{"id": "tx40ya", "name": "2balkan4you", "description": "In memory of r/2balkan4you banned by u/Chtorrr in the beginning 2022, a subreddit where people from the Balkans could joke together.\n\nMany new subreddits have tried to reconstruct the community of around 200k members, but the majority of have become empty.", "links": {"website": ["https://subredditstats.com/r/2balkan4you"], "subreddit": ["balkans_irl"]}, "path": {"56-166, T": [[1918, 322], [1918, 334], [1936, 334], [1936, 322]]}, "center": {"56-166, T": [1927, 328]}},
```

Comment:
"majority of **them** have become empty"

Reply:
Implemented fix.

Thanks for the catch!